configure.ac

```diff
@@ -480,6 +497,10 @@ dnl AC_CHECK_HEADERS([stdint.h whcar.h malloc.h])
 # mandatory:
 AC_CHECK_HEADERS([sys/types.h signal.h stddef.h], [],
 	[AC_MSG_ERROR([mandatory header could not be found or included])])
+AC_CHECK_HEADERS([jni.h], [],
+  [AC_MSG_ERROR([jni.h is required but could not be found])])
+AC_CHECK_HEADERS([jni_md.h], [],
+  [AC_MSG_ERROR([jni_md.h is required but could not be found])])
```

Comment:
We can leave this as-is for now, later on it should only be done if `--with-java` is specified.
You may opt to do this part after your mid-term evaluation, whenever you do check how `--with-xml2` is handled.